internal/datanode/flow_graph_manager.go

```diff
@@ -0,0 +1,127 @@
+// Licensed to the LF AI & Data foundation under one
+// or more contributor license agreements. See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership. The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package datanode
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/milvus-io/milvus/internal/log"
+	"github.com/milvus-io/milvus/internal/proto/datapb"
+
+	"go.uber.org/zap"
+)
+
+type flowgraphManager struct {
+	flowgraphs sync.Map // vChannelName -> dataSyncService
+}
+
+func newFlowgraphManager() *flowgraphManager {
+	return &flowgraphManager{}
+}
+
+func (fm *flowgraphManager) AddAndStart(dn *DataNode, vchan *datapb.VchannelInfo) error {
```

Comment:
If this struct is only used in package datanode, we could use lower case to name the function

Reply:
And I suggest to change the name to `add`